.rhdh/docker/Dockerfile

```diff
@@ -51,17 +51,18 @@ RUN chmod +x "$YARN"
 
 # Stage 2 - Install dependencies
 COPY $EXTERNAL_SOURCE_NESTED/dynamic-plugins/ ./dynamic-plugins/
```

Comment:
I think @schultzp2020 would prefer if we don't do this line and instead DO include the dynamic-plugins/ folders in update-Dockerfile.sh line 4

Reply:
Do we remove `$EXTERNAL_SOURCE_NESTED/dynamic-plugins/ ./dynamic-plugins/` entirely, or does he prefer that we keep it and still copy it's package.json files?